local-cli/util/parseCommandLine.js

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
parseCommandLine is used in a few places internally at FB.
Also it is used in dependencies.js, so it needs not to be removed.